engine/src/main/java/org/terasology/world/block/family/RotationBlockFamily.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.world.block.family;
+
+import org.terasology.math.Rotation;
+import org.terasology.math.Side;
+import org.terasology.world.block.Block;
+
+public interface RotationBlockFamily {
+    Block getRolledClockwise(Block currentBlock, Side relativeSide);
```

Comment:
Also, is "relativeSide" the side around which the clockwise rotation is performed? I wonder if it could be renamed to something less ambiguos, i.e. "axis"?